lib/amqp/queue.rb

```diff
@@ -145,10 +145,14 @@ class Queue
     # @return [AMQP::Channel]
     attr_reader :channel
 
-    # @return [Array<Hash>] All consumers on this queue.
+    # @return [Array<!-- <Hash> -->] All consumers on this queue.
```

Comment:
Is this a new YARD syntax?

Reply:
Sorry, I don't know how this got in there (although I have an idea how I must have left it there). Removed.